pkg/agent/controller/traceflow/packetin.go

```diff
@@ -344,7 +348,16 @@ func (c *Controller) parsePacketIn(pktIn *ofctrl.PacketIn) (*crdv1beta1.Traceflo
 			}
 			ob.Action = crdv1beta1.ActionForwardedOutOfOverlay
 		} else if outputPort == gwPort { // noEncap
-			ob.Action = crdv1beta1.ActionForwarded
+			isPodIP := false
```

Comment:
Deleted Ran's comment accidentally. cc @gran-vmv 
repost here: 
`Do we support Egress on Noencap mode? if yes, should we merge the condition with above? `

I don't think we support Egress in NoEncap mode, but it would be supported in hybrid mode. It seems SupportsEncap() will be true when it's Hybrid mode, so these two can be merged? cc @hongliangl @tnqn

Reply:
+1 to handle it as a separate PR

Reply:
Aded TODO comment.

Reply:
> I don't think we support Egress in NoEncap mode, but it would be supported in hybrid mode. It seems SupportsEncap() will be true when it's Hybrid mode, so these two can be merged?

We don't have a plan to support Egress in NoEncap mode, but in hybrid mode, for those Nodes which are in the same subnet, they are actually "noEncap" mode to each other.

Reply:
> > I don't think we support Egress in NoEncap mode, but it would be supported in hybrid mode. It seems SupportsEncap() will be true when it's Hybrid mode, so these two can be merged?
> 
> We don't have a plan to support Egress in NoEncap mode, but in hybrid mode, for those Nodes which are in the same subnet, they are actually in "noEncap" mode to each other.

That will change existing released behavior, and this PR only focused on noEncap mode, thus I added TODO comment here.